changelog.md

```
@@ -1,5 +1,10 @@
 # Changelog
 
+### [6.12.2] 2026-08-05
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)